src/plugins/intel_cpu/src/emitters/snippets/x64/kernel_executors/brgemm.cpp

```diff
@@ -63,11 +63,22 @@ bool BrgemmKernelConfig::operator==(const BrgemmKernelConfig& rhs) const {
 }
 
 void BrgemmKernelConfig::update(dnnl_dim_t M, dnnl_dim_t N, dnnl_dim_t K, dnnl_dim_t LDA, dnnl_dim_t LDB, dnnl_dim_t LDC) {
-    m_M = M; m_N = N; m_K = K;
-    m_LDA = LDA; m_LDB = LDB; m_LDC = LDC;
+    // If M is zero, it means that Brgemm won't be executed (in Loop with work_amount = 0, for example)
+    // To process this case, we have to make this Config as empty (nullify runtime parameters)
+    if (M == 0 && !utils::one_of(0, N, K, LDA, LDB, LDC)) {
+        m_M = 0; m_N = 0; m_K = 0;
+        m_LDA = 0; m_LDB = 0; m_LDC = 0;
+    } else {
+        m_M = M; m_N = N; m_K = K;
+        m_LDA = LDA; m_LDB = LDB; m_LDC = LDC;
+    }
     m_hash = compute_hash();
 }
 
+bool BrgemmKernelConfig::is_empty() const {
+    return everyone_is(0, m_M, m_N, m_K, m_LDA, m_LDB, m_LDC);
```

Comment:
Honestly, We'll get confused if we don't have empty config state. Because it's not clear when config is valid, when is not valid. Let's discuss offline this moment.